superset/db_engine_specs/ocient.py

```diff
@@ -154,6 +160,12 @@ class OcientEngineSpec(BaseEngineSpec):
     force_column_alias_quotes = True
     max_column_name_length = 30
 
+    # Store mapping of superset Query id -> Ocient ID
+    # These are inserted into the cache when executing the query
+    # They are then removed, either upon cancellation or query completion
+    query_id_mapping: Dict[str, str]= dict()
```

Comment:
(uber nit): do you have a way to format your code?

Reply:
i.e. are you manually adding in whitespace etc?

Reply:
Right now, manually. We could probably involve something like black.

Reply:
(yes, please format your code with black, see superset contribution guidelines => sqlalchemy_ocient/sqlalchemy_ocient.py)

Reply:
https://github.com/apache/superset/blob/master/CONTRIBUTING.md#python

Reply:
You can use either the vscode extension or install and format manually. They also supply a commit hook if you'd like to do that